platform-common/src/main/java/com/softicar/platform/common/ui/image/Images.java

```diff
@@ -0,0 +1,69 @@
+package com.softicar.platform.common.ui.image;
+
+import com.softicar.platform.common.core.exceptions.SofticarException;
+import com.softicar.platform.common.core.exceptions.SofticarIOException;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+import javax.imageio.ImageIO;
+
+/**
+ * Convenience methods for {@link ImageIO}.
```

Comment:
- If this class is indeed ImageIO specific - should we reflect that in the class name? e.g. `ImageIOImages` or so
- Either way, I think we should explicitly mention that supported formats depend on what's supported by `ImageIO`.

Reply:
`ImageIO` is an integral part of the Java API, IMO, everybody working with images in Java will use it by default.

Reply:
Okay